tests/rptest/e2e_tests/flink_scale_test.py

```diff
@@ -82,54 +88,140 @@ def assert_failed_jobs(job_list):
                 "Flink reports fails for " \
                 f"transaction workload:\n{desc}"
 
-        # TODO: Validate/Gather info on
-        # - Transactions can be generated per one flink job
-        # - Transactions per node
-        # - Transactions total >500K
+        def get_hwm_for_topic(topic):
+            _hwm = 0
+            for partition in self.rpk.describe_topic(topic):
+                # Add currect high watermark for topic
+                _hwm += partition.high_watermark
+            return _hwm
```

Comment:
I'd rename this, hwm is per partition, its odd to have a hwm for topic, rename to total_event_count or something?

Reply:
Agreed.